modules/platform/nuxeo-platform-task/nuxeo-platform-task-core/src/main/java/org/nuxeo/ecm/platform/task/core/service/TaskServiceImpl.java

```diff
@@ -482,9 +482,15 @@ public String getParentPath() {
         return tasks;
     }
 
+
+    @Override
+    public void reassignTask(CoreSession session, String taskId, List<String> actors, String comment) {
+        reassignTask(session, taskId, actors, comment, new HashMap<>());
```

Comment:
Can you do that as a `default` method in the interface instead of in the implementation? That's usually cleaner for "things that delegate for backward compat". (Same for `delegateTask`.)